source/_integrations/reolink.markdown

```diff
@@ -250,6 +251,8 @@ Depending on the supported features of the camera, switch entities are added for
 - PIR reduce false alarm*
 - Chime LED
 
+When the **Privacy mode** is ON, allmost all other entities will be unavailable because the camera shuts down the API and camera streams. When turning OFF the **Privacy mode**, all entities will become available again. Take this into consideration when making automations; ensure the **Privacy mode** is OFF before changing camera settings using other entities.
```

Comment:
```suggestion
When the **Privacy mode** is ON, almost all other entities will be unavailable because the camera shuts down the API and camera streams. When turning OFF the **Privacy mode**, all entities will become available again. Take this into consideration when making automations; ensure the **Privacy mode** is OFF before changing camera settings using other entities.
```